compiler/ast/reports.nim

```diff
@@ -759,9 +760,16 @@ static:
     echo "parser reports   = ", len(repParserKinds)
     echo "internal reports = ", len(repInternalKinds)
 
-let reportEmpty* = Report(
+const
+  emptyReportId* = ReportId(0)
+    ## report id for the `emptyReport`
+
+let emptyReport* = Report(
```

Comment:
If report ID is used directly for the free operation, explicit empty report won't be necessary